server/routes/index.ts

```diff
@@ -17,12 +17,15 @@ import populateCurrentUser from '../middleware/populateCurrentUser'
 import flashMessages from '../middleware/flashMessageMiddleware'
 import fromReviewMiddleware from '../middleware/fromReviewMiddleware'
 import PrisonerController from './prisonerController'
+import WhatsNewPage from './whatsNewPage/whatsNewPage'
 
 export default function Index(services: Services): Router {
   const router = Router({ mergeParams: true })
   const prisonerController = new PrisonerController(services.prisonerService)
+  const whatsNewPage = new WhatsNewPage()
 
   router.get('/prisoner/:nomsId/image', prisonerController.getImage())
+  router.get('/whats-new-page', whatsNewPage.GET)
```

Comment:
These should be authenticated right? 

seems like this would be better belonging in the `homeRoutes` section?
This tends to have things that are more meta - accessiblity statements/ ts+cs etc..